.github/workflows/build-astro-nvim.yml

```
@@ -21,9 +26,97 @@
+            source ~/.bashrc 2>/dev/null || true
+            go version && 
+            echo 'Go version check: PASSED' || 
+            (echo 'Go version check: FAILED' && exit 1)
+          "
+
+      - name: Test Node.js installation
+        run: |
+          echo "Testing Node.js installation..."
+          docker run --rm --entrypoint="" ${{ vars.DOCKER_USERNAME }}/${{ env.IMAGE_NAME }}:latest bash -c "
+            source ~/.bashrc 2>/dev/null || true
+            node --version && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)